source/features/linkify-code.tsx

```diff
@@ -20,23 +20,27 @@ function initTitle(signal: AbortSignal): void {
 }
 
 function linkifyContent(wrapper: Element): void {
+	// Mark code block as touched to avoid `shorten-links` from acting on these new links in code
+	wrapper.classList.add(linkifiedURLClass);
+
 	const errors = linkifyURLs(wrapper);
 	if (errors) {
 		features.log.error(import.meta.url, 'Links already exist');
 		console.log(errors);
 	}
 
-	// Linkify issue refs in comments, exclude gists:
-	// https://github.com/refined-github/refined-github/pull/3844#issuecomment-751427568
-	if (!pageDetect.isGist()) {
-		const currentRepo = getRepo() ?? {};
-		for (const element of $$('.pl-c', wrapper)) {
-			linkifyIssues(currentRepo, element);
-		}
+	const currentRepo = pageDetect.isGlobalSearchResults()
+		? getRepo(wrapper.parentElement!.querySelector('.blob-num a')!.href)
```

Comment:
Are you sure this works? The `wrapper` seems to be a `td`, so the parentElement is a `tr`, which does not contain any links.

<img width="492" alt="Screenshot 14" src="https://github.com/refined-github/refined-github/assets/1402241/2bb73ba7-1836-4b42-b7d0-33991541e95a">


The right ancestor is probably `Box-sc-g0xbh4-0 kNKjla`, which is hard to target 😬

Reply:
The selector targets the line link, rather than the repo link at the top of each file.

`tr (parent) .blob-num a` seems to work properly, seeing as the hovercard shows (screenshot ^)

![image](https://github.com/refined-github/refined-github/assets/58778985/f442eba3-cbc7-48ea-b9f6-60be9b29e1dc)

Reply:
Oh neat, I thought it was targeting the file header